src/connectors/pagination/connectPagination.js

```diff
@@ -123,14 +136,42 @@ export default function connectPagination(renderFn, unmountFn) {
         return maxPages !== undefined ? Math.min(maxPages, nbPages) : nbPages;
       },
 
+      getPages({ nbPages, currentRefinement }) {
```

Comment:
Where is this function used?

Reply:
nowhere, was an artefact from an older version of this PR